metaseq/tasks/streaming_language_modeling.py

```diff
@@ -246,6 +281,10 @@ def _check_cm3_parameterization(self):
 
     def _create_cm3_special_tokens(self):
         self.cm3_sentinel_end = "<eoss>"
+        self.cm3_break = "<racm3:break>"
+        self.dictionary.add_symbol(self.cm3_break)
```

Comment:
It's all looking great. 
We want to make a change here to recycle unused embedding index of cm3_break and sentinel for the next version. 
Should i just add a commit on top of this PR? Or do I file a different PR?